build.sbt

```diff
@@ -17,16 +17,15 @@
 import Dependencies._
 import com.github.sbt.jacoco.report.JacocoReportSettings
 
-ThisBuild / name := "fa-db"
-ThisBuild / organization := "za.co.absa.fa-db"
+ThisBuild / organization := "za.co.absa"
```

Comment:
why you dropped the `name` attribute?

Reply:
Pointless variable introduced by mistake